src/cow-react/modules/limitOrders/hooks/useIsSellOrder.ts

```diff
@@ -0,0 +1,12 @@
+import { useMemo } from 'react'
+import { useAtomValue } from 'jotai'
+
+import { OrderKind } from '@cowprotocol/contracts'
+import { limitOrdersAtom } from '@cow/modules/limitOrders/state/limitOrdersAtom'
+
+// Returns boolean if the current order kind is SELL or BUY
+export function useIsSellOrder(): boolean {
+  const { orderKind } = useAtomValue(limitOrdersAtom)
+
+  return useMemo(() => orderKind === OrderKind.SELL, [orderKind])
```

Comment:
What about when orderKind is `null`?
Then it would be like it's a BUY order which is not what we want most of the cases.

I would think it's safer to default it to SELL